scripts/allowlist/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "create_qbft_genesis_content",
+  "version": "1.0.0",
+  "main": "createContent.js",
```

Comment:
`main` is only for modules and only when doing something non-standard (cf. https://docs.npmjs.com/cli/v7/configuring-npm/package-json/#main). We don't need it.

Reply:
done